### PR TITLE
Ka 2018 01 add idea identifier

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

File: meinberlin/apps/exports/mixins.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class ItemExportWithReferenceNumberMixin(VirtualFieldMixin):
+
+    def get_virtual_fields(self, virtual):
+        if 'reference_number' not in virtual:
+            virtual['reference_number'] = _('Reference Number')
+        return super().get_virtual_fields(virtual)
+
+    def get_category_data(self, item):
```
Comment (MagdaN) [inline meinberlin/apps/exports/mixins.py]: method should be called `get_reference_number_data` i guess

**Comment by kleingeist** [comment]
wuhu nice work :)
but i have to look at it tomorrow morning. my concentration is way to low to get an overview over that many files

File: meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_detail.html [hunk 2]
```diff
@@ -34,11 +34,10 @@ <h2 class="moderator-statement__title">{% trans 'Official Statement' %}</h2>
 {% endblock %}
 
 {% block dropdown_items %}
-    {% get_item_permission object 'moderate' as moderator_perm %}
-    {% has_perm moderator_perm request.user proposal as is_moderator %}
+    {% has_perm 'meinberlin_budgeting.moderate_proposal' request.user proposal as is_moderator %}
     {% if is_moderator %}
         <li>
-            {% get_item_url object 'moderate' as moderate_url %}
+            {% url 'meinberlin_budgeting:proposal-moderate' year=object.created.year pk=object.pk as moderate_url %}
```
Comment (kleingeist) [inline meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_detail.html]: i would skip the assignment and just generate the url directly in the next line

File: meinberlin/apps/kiezkasse/templates/meinberlin_kiezkasse/proposal_detail.html [hunk 3]
```diff
@@ -34,11 +34,10 @@ <h2 class="moderator-statement__title">{% trans 'Official Statement' %}</h2>
 {% endblock %}
 
 {% block dropdown_items %}
-    {% get_item_permission object 'moderate' as moderator_perm %}
-    {% has_perm moderator_perm request.user proposal as is_moderator %}
+    {% has_perm 'meinberlin_kiezkasse.moderate_proposal' request.user proposal as is_moderator %}
     {% if is_moderator %}
         <li>
-            {% get_item_url object 'moderate' as moderate_url %}
+            {% url 'meinberlin_kiezkasse:proposal-moderate' year=object.created.year pk=object.pk as moderate_url %}
```
Comment (kleingeist) [inline meinberlin/apps/kiezkasse/templates/meinberlin_kiezkasse/proposal_detail.html]: see above: skipping asignment

**Comment by kleingeist** [comment]
meh i hopped we could get rid of the url related item tags :( 
but i just realized we still have to use get_item_create_url, get_item_update_url and get_item_delete_url
but we could delete get_item_detail_url to encourage the use of item.get_absolute_url :D

**Pull request merged**